x-pack/plugins/maps/public/classes/layers/new_vector_layer_wizard/utils/indexing_service.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { http as httpService } from './http_service';
+import { getSavedObjectsClient } from '../../../../kibana_services';
+
+export const getExistingIndexNames = async () => {
+  const indexes = await httpService({
+    url: `/api/index_management/indices`,
```

Comment:
`/api/index_management/indices` is the only I didn't change over since it's a one-off reference to the index management api. All others referring to maps api endpoints have been changed over to use the correct constants